bootstrap/tracing.go

```diff
@@ -165,3 +241,22 @@ func Merge(maps ...map[string]any) map[string]any {
 
 	return merged
 }
+
+func (b *Bootstrap) toAttributes(extras map[string]any) []attribute.KeyValue {
```

Comment:
This returns otel specific attributes right? Maybe this could be called `toOpenTelemetryAttributes` or something to be clear it's only for one provider?